client/pages/home.jsx

```diff
@@ -1,10 +1,71 @@
 import React from 'react';
 import AppContext from '../lib/app-context';
+import Redirect from '../components/redirect';
 
 export default class Home extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      projects: [],
+      projectSelected: false,
+      mounted: false
+    };
+    this.handleProjectView = this.handleProjectView.bind(this);
+  }
+
+  handleProjectView() {
+    this.setState({ projectSelected: !this.state.projectSelected });
+  }
+
+  componentDidMount() {
+
+    fetch('/api/home/project', { method: 'GET' })
+      .then(res => res.json())
+      .then(result => {
+        this.setState({ projects: this.state.projects.concat(result) });
+      });
+    this.setState({ mounted: true });
+  }
+
   render() {
+    const projectList = this.state.projects.map(project =>
+      <div key={project.projectId} >
+        {
+          <div onClick={this.handleProjectView} className="card margin-top-1x">
+            <div className="row d-flex align-items-center card-body">
+              <i className="fas fa-lightbulb fa-3x margin-1x icon-color"></i>
+              <div>
+                <h2 className="homepage-font">{project.projectName}</h2>
+              </div>
+            </div>
+          </div>
+        }
+      </div>
+    );
+
+    if (this.state.projectSelected) {
+      return <Redirect to="project" />;
+    }
```

Comment:
Instead of redirecting, you'll need to have another view to render in your App component. Each screen should be chosen by App.